build.sbt

```diff
@@ -13,3 +13,12 @@ scalacOptions in Test ++= Seq("-deprecation")
 testOptions in Test += Tests.Argument("-Djava.library.path=target/native")
 
 publishTo := Some(Resolver.file("file",  new File( "releases" )) )
+
+lazy val packageSqlite3C = taskKey[Unit]("Packages only Sqlite3C.java")
+
+packageSqlite3C := {
+  (compile in Compile).value
+  val cmd = Seq("jar", "-cf", s"target/scala-2.11/sqlite3c-${version.value}.jar",
```

Comment:
The comment was lost, but in case you want to make this more portable:

`classDirectory.value` translates to "target/scala-2.11/classes"
`crossTarget.value` translates to "target/scala-2.11"